payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/AfterpayClearpayHeaderElementTest.kt

```diff
@@ -13,15 +14,35 @@ import java.util.Locale
 class AfterpayClearpayHeaderElementTest {
 
     @Test
-    fun `Verify label is correct`() {
+    fun `Verify label is correct for USD`() {
```

Comment:
Can we add a test for image first? As in one of the images you posted in the PR description

Reply:
We can't do this on the main sheet because the order of the LPMs in the selectors is not consistent.  As we advance our screenshot testing I think it would be good to add testing for the core UI components like the Html element rather than the payment method specific UI.

Reply:
I am not really following. Can't we change the locale to korean or something that makes the image icon tag first in the string? Not looking for a screenshot test, an assert as you have in this test would suffice